DeliveryServiceModel/Order.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+
+namespace DeliveryServiceModel
+{
+    public class Order
+    {
+        public int Id { get; init; }
+
+        public User User { get; set; }
+
+        public ICollection<Appliance> Basket { get; set; }
```

Comment:
maybe rename?
backet it is before Order
when i make order it is already in Order
it is not basket
up to you